### PR TITLE
[14.0][MIG] hr_org_chart_overview

### PR DESCRIPTION
**Pull request by aphon61bank**
Add t-call-assess and t-css for run code, Ready for reviwe.

### CODE REVIEW

File: hr_org_chart_overview/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "HR Org Chart Overview",
+    "version": "14.0.1.1.0",
```
Comment (LoisRForgeFlow) [inline hr_org_chart_overview/__manifest__.py:6]: just `14.0.1.0.0`

**Review by LoisRForgeFlow — COMMENTED** [review]
@aphon61bank Could you clean the commit history and remove the last merge commit? I would also be great if you could split between what are changes from pre-commit and what is the actual migration (see [the migration wiki](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#technical-method-to-migrate-a-module-from-130-to-140-branch).

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
I miss the navigation bar from v13:
image

This is current v14 in runbot:
image

**Comment by dreispt** [comment]
Superseded by #953

**Pull request closed without merging**